druid/examples/flex.rs

```diff
@@ -13,17 +13,39 @@
 // limitations under the License.
 
 //! Demonstrates alignment of children in the flex container.
+//! This is more of a widget showcase where you can turn all the knobs
+//! and see how it behaves and less of an example.
 
 use druid::widget::prelude::*;
 use druid::widget::{
     Button, Checkbox, CrossAxisAlignment, Flex, Label, MainAxisAlignment, ProgressBar, RadioGroup,
     SizedBox, Slider, Stepper, Switch, TextBox, WidgetExt,
 };
-use druid::{
-    AppLauncher, Color, Data, Lens, LensExt, LocalizedString, PlatformError, WidgetId, WindowDesc,
-};
+use druid::{AppLauncher, Color, Data, Lens, LensExt, WindowDesc};
 
 const DEFAULT_SPACER_SIZE: f64 = 8.;
+const SPACER_OPTIONS: [(&str, Spacers); 4] = [
+    ("None", Spacers::None),
+    ("Default", Spacers::Default),
+    ("Flex", Spacers::Flex),
+    ("Fixed:", Spacers::Fixed),
+];
+const MAIN_AXIS_ALIGNMENT_OPTIONS: [(&str, MainAxisAlignment); 6] = [
+    ("Start", MainAxisAlignment::Start),
+    ("Center", MainAxisAlignment::Center),
+    ("End", MainAxisAlignment::End),
+    ("Between", MainAxisAlignment::SpaceBetween),
+    ("Evenly", MainAxisAlignment::SpaceEvenly),
+    ("Around", MainAxisAlignment::SpaceAround),
+];
+const CROSS_AXIS_ALIGNMENT_OPTIONS: [(&str, CrossAxisAlignment); 4] = [
+    ("Start", CrossAxisAlignment::Start),
+    ("Center", CrossAxisAlignment::Center),
+    ("End", CrossAxisAlignment::End),
+    ("Baseline", CrossAxisAlignment::Baseline),
+];
+const FLEX_TYPE_OPTIONS: [(&str, FlexType); 2] =
+    [("Row", FlexType::Row), ("Column", FlexType::Column)];
```

Comment:
This is much neater IMO. :smile:

Reply:
yeah having them inline wasn't very nice, just a shame it cant be a `Vec` but so be it.